src/drt/src/TritonRoute.cpp

```diff
@@ -439,6 +439,22 @@ void TritonRoute::readParams(const string& fileName)
   }
 }
 
+void TritonRoute::addUserSelectedVia(const std::string& viaName)
+{
+  if (db_->getChip() == nullptr || db_->getChip()->getBlock() == nullptr
+      || db_->getTech() == nullptr) {
+    logger_->error(DRT, 610, "Load desing before setting default vias");
```

Comment:
I suspect this should be "Load design ..."

Reply:
nice catch :smile: